cmake/FindLIBHWLOC.cmake

```diff
@@ -46,13 +46,6 @@ endif()
 
 if(LIBHWLOC_LIBRARY)
     message(STATUS "  Found libhwloc using find_library()")
-    message(STATUS "    LIBHWLOC_LIBRARIES = ${LIBHWLOC_LIBRARIES}")
-    message(STATUS "    LIBHWLOC_INCLUDE_DIRS = ${LIBHWLOC_INCLUDE_DIRS}")
-    message(STATUS "    LIBHWLOC_LIBRARY_DIRS = ${LIBHWLOC_LIBRARY_DIRS}")
-    message(STATUS "    LIBHWLOC_API_VERSION = ${LIBHWLOC_API_VERSION}")
-    if(WINDOWS)
-        message(STATUS "    LIBHWLOC_DLL_DIRS = ${LIBHWLOC_DLL_DIRS}")
-    endif()
 
```

Comment:
hmm.. with this approach, if we get to any of the FATAL_ERRORs below we won't get these STATUS messages, so we won't know what failed, I guess...